storage/storage.go

```diff
@@ -253,11 +253,10 @@ type TupleBackend interface {
 		opts PaginationOptions,
 	) ([]*openfgapb.Tuple, []byte, error)
 
-	// Write updates data in the tuple backend, performing all delete operations in
-	// `deletes` before adding new values in `writes`, returning the time of the transaction, or an error.
-	// It is expected that
-	// - there is at most 10 deletes/writes
-	// - no duplicate item in delete/write list
+	// Write updates data in the tuple backend, performing all delete operations in `deletes` before writing the values
+	// in `writes`, returning the time of the transaction, or an error. It is expected that there are at most
```

Comment:
We don't return the time of the transaction in this method signature. Let's omit that bit.